api/capi/src/nnstreamer-capi-single-new.c

```diff
@@ -478,10 +471,10 @@ ml_single_close (ml_single_h single)
   ML_SINGLE_GET_VALID_HANDLE_LOCKED (single_h, single, 1);
 
   single_h->join = TRUE;
-  pthread_cond_broadcast (&single_h->cond);
+  g_cond_broadcast (&single_h->cond);
 
   ML_SINGLE_HANDLE_UNLOCK (single_h);
-  pthread_join (single_h->thread, NULL);
+  g_thread_join (single_h->thread);
```

Comment:
plz consider to add g_mutex_clear() and g_cond_clear() before freeing single-handle.

Reply:
updated.